app/src/main/java/com/yeonae/chamelezone/data/source/remote/member/MemberRemoteDataSourceImpl.kt

```diff
@@ -191,37 +188,94 @@ class MemberRemoteDataSourceImpl private constructor(private val memberApi: Memb
     override fun findPassword(
         email: String,
         phone: String,
-        callBack: MemberCallBack<MemberResponse>
+        callBack: MemberCallBack<FindPasswordResponse>
     ) {
         val jsonObject = JsonObject().apply {
             addProperty(EMAIL, email)
             addProperty(PHONE, phone)
         }
         memberService.findPassword(jsonObject)
-            .enqueue(object : Callback<MemberResponse> {
-                override fun onFailure(call: Call<MemberResponse>, t: Throwable) {
+            .enqueue(object : Callback<FindPasswordResponse> {
+                override fun onFailure(call: Call<FindPasswordResponse>, t: Throwable) {
                     Log.e("tag", t.toString())
                 }
 
                 override fun onResponse(
-                    call: Call<MemberResponse>,
-                    response: Response<MemberResponse>
+                    call: Call<FindPasswordResponse>,
+                    response: Response<FindPasswordResponse>
                 ) {
                     if (response.code() == SUCCESS) {
                         response.body()?.let { callBack.onSuccess(it) }
+                    } else if (response.code() == REQUEST_ERR) {
+                        callBack.onFailure(App.instance.context().getString(R.string.information_not_exist))
                     }
                 }
 
             })
     }
 
+    override fun checkSecurityCode(
+        securityCode: String,
+        email: String,
+        phone: String,
+        callBack: MemberCallBack<SecurityCodeResponse>
+    ) {
+        val jsonObject = JsonObject().apply {
```

Comment:
```suggestion
        val requestBody = JsonObject().apply {
```
의미가 담겨있지 않은 jsonObject라는 네이밍보다는, 이런 네이밍을 생각해서 변경해보는것은 어떤가요?🤔

Reply:
수정했습니당